File: homework_1/3.min_of_four/min_of_four.cpp
#include <iostream>
using namespace std;

int main(){
    int x, y, z, a, min;
    cin >> x >> y >> z >> a;

    if(x + y + z + a == 0){
        min = x;    
        if(y < min)
            min = y;
        if(z < min)
            min = z;
        if(a < min)
            min = a;
        
        cout << min << endl;
    }

    return 0;
}

File: homework_1/3.min_of_four/min_of_four.js
const prompt = require("prompt-sync")();

const x = +prompt();
const y = +prompt();
const z = +prompt();
const a = +prompt();

if(x + y + z + a == 0){
    let min = x;    
    if(y < min)
        min = y;
    if(z < min)
        min = z;
    if(a < min)
        min = a;
    
    console.log(min);
}